LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 AitorDB
```

Comment:
I believe we should keep the original copyright and simply add ours after it (also note the project name change).

Reply:
Good point.

Reply:
Indeed. I also kept the original copyright when I've setup my fork to publish in HACS.
But there is another file where is @AirtonDB username and email as author. I think we should keep his name, but probably remove his email address as I  belive he archived his repo due to no intention to offer support to it anymore.